### PR TITLE
HT-2588 - Remove PDUS access for US

### PR DESCRIPTION
**Pull request by respinos**
**WIP**

Think we're colliding --- 

- I like the way you set up the fixtures, but of course those aren't available on grog :-(
- I'm trying to get the tests runnable on `grog`
- anyway ht_affiliate.t laboriously lists all the attributes and responses for ht_affiliate
- ht_affiliate.pl mocks what it needs so no database is needed

`rights.t` won't run on `grog` yet (you can see my tweak to `namespaces.t`); maybe it could use the "skip" features to only run locally?

### CODE REVIEW

**Comment by aelkiss** [comment]
Regarding running the tests on grog, we could certainly set up a database with the fixtures that it can use (on mysql-htdev). Let me know if you need me to set something up.

File: t/ht_affiliate.t [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+    my $ar = Access::Rights->new($C, $id);
+    my $status = $ar->check_final_access_status($C, $id);
+    return $status;
+}
+
+my $num_tests = 0;
+
+# US institution
+is(test_attr(1, 1), 'allow', 'ht_affiliate + attr=1 + source=1'); $num_tests += 1;
```
Comment (aelkiss) [inline t/ht_affiliate.t]: I'm not wild about seeing magic numbers here. The least-intrusive modification might be to define e.g. `$ATTR_PD` in `RightsGlobals.pm`. I doubt the current rights model will stick around as-is long term, so I don't think putting in the effort to dynamically load the id -> name mapping from the `attributes` table would be worthwhile now.

Comment (respinos) [reply t/ht_affiliate.t]: I added some reverse maps in `RightsGlobals.pm` and updated the script to use the labels.

**Review by aelkiss — APPROVED** [review]
Probably would be good to squash commits, and see the question about `umall`, but otherwise looks good!

**Review by moseshll — COMMENTED** [review]
I've got a proof of concept that can run t/rights.t on grog and in Docker by asking rights_current for a "real" example volume based on the attr_reason_src_profile string values. I'm sure it's dreadfully inefficient, but with the right environment variables set it looks OK when run against this branch: 

https://gist.github.com/moseshll/ad80ed651d7fc7bf1a8541a1685cb303

Not sure anybody but me will like that approach as it seems clunky.

**Review by moseshll — APPROVED** [review]

**Comment by respinos** [comment]
@moseshll we should probably work out if access tests should be programs or a boring list of statements. Probably we should be doing something with the access review spreadsheets.

**Pull request merged**